src/EnergyPlus/WaterThermalTanks.hh

```diff
@@ -93,14 +93,16 @@ namespace WaterThermalTanks {
     // order of ControlTypeEnum and PriorityEnum must be enforced
     enum ControlTypeEnum
     {
-        Cycle,
+        Cycle = 0,
         Modulate
     };
 
     // order of ControlTypeEnum and PriorityEnum must be enforced
+    // WaterThermalTankData uses the same int ControlType to assign either ControlTypeEnum (WaterHeater:Mixed) or PriorityEnum
+    // (WaterHeater:Stratified), so in order to avoid problems, start the int value here as 2 so they don't risk collapsing.
     enum PriorityEnum
     {
-        MasterSlave, // water heater only, master-slave priority control of heater elements
+        MasterSlave = 2, // water heater only, master-slave priority control of heater elements
         Simultaneous // water heater only, simultaneous control of heater elements
     };
```

Comment:
Avoid future collapses. (We should perhaps make a habit of using enum classes instead...)

Reply:
Yes, we should definitely do enum classes.  But I appreciate that that change is likely out of scope for this little defect fix.